src/targets/node/nodeSourcePathResolver.ts

```diff
@@ -25,7 +26,7 @@ export class NodeSourcePathResolver extends SourcePathResolverBase<IOptions> {
       return '';
     }
 
-    const withBase = path.resolve(this.options.basePath, this.applyPathOverrides(url));
```

Comment:
This was causing some cross-platform failures. `properResolve` joins and normalizes to the OS of the base path.